modules/core/04-channel/keeper/keeper_test.go

```diff
@@ -105,13 +106,13 @@ func (suite *KeeperTestSuite) TestGetAllChannelsWithPortPrefix() {
 			name:             "transfer channel is retrieved with prefix",
 			prefix:           "tra",
 			allChannels:      allChannels,
-			expectedChannels: []types.IdentifiedChannel{types.NewIdentifiedChannel(transfertypes.TypeMsgTransfer, ibctesting.FirstChannelID, types.Channel{})},
+			expectedChannels: []types.IdentifiedChannel{types.NewIdentifiedChannel(MsgTransfer, ibctesting.FirstChannelID, types.Channel{})},
```

Comment:
I think you can use either `transfertypes.PortID` or even [`ibcmock.PortID`](https://github.com/cosmos/ibc-go/blob/main/testing/mock/mock.go#L25) for the first argument (portID) here instead, and remove the `MsgTransfer` const above.

Reply:
Thanks for the review! Will update this